app/controllers/register/trustees/individual/TrusteeAddressController.scala

```diff
@@ -68,7 +68,7 @@ class TrusteeAddressController @Inject()(
               postCall(mode, Index(index), srn),
```

Comment:
Please use toggle for TrusteeNameId vs TrusteeDetailsId

Reply:
added condition for H&S, please check now